### ISSUE TITLE
Is Knex still maintained?

### ISSUE THREAD

**Issue by DjangoCrema** [knex/knex#6154]
Is the project still alive? I see the last release was a year ago. At this moment, it has more than 60 deprecated dependencies.

**Comment by KristjanTammekivi** [comment]
https://github.com/kysely-org/kysely looks pretty good, I've made a small pull request there in the past. I haven't actually tried it out yet in a real project, but I might now considering I have 3 open issues and 2 open PRs here

**Comment by HaTiIn** [comment]
@kibertoad ?

**Comment by kibertoad** [comment]
I'm still active, I just struggle to find time and energy to spend on knex specifically, especially since drizzle, kysely and prisma are all generally better options these days.

That said, I definitely should release a new version with dependencies fixed. Please stay tuned.

**Comment by chenbimo** [comment]
I like this, hope knex release next version.

**Comment by jpike88** [comment]
> drizzle, kysely and prisma are all generally better options these days

None of them support the mutable query building approach knex has. It's allowed us to make highly customized queries without a million abstractions to handle it. I think knex has it's place forever, and we certainly wouldn't consider any of the options above as suitable alternatives for what we do, unless you're talking about very simple, constrained models without complex joins involved.

**Comment by electriquo** [comment]
@kibertoad Is there anyway we could assist?

**Comment by rubengmurray** [comment]
Yeah I'm keen to see `knex` maintained even if just dependency updates and minor updates.

Long-term though it feels like this package is being sunsetted?

**Comment by mercmobily** [comment]
Nope we are going through issues and PRs, as well as paths for code improvements and modernisation. Knex is too good to be sunsetted :D 

@electriquo I am organising a group of people willing to lend a hand to make sure knex is in active maintenance. Get in touch with me if you are willing to help!

**Comment by rubengmurray** [comment]
Great to hear 🙏  I could be interested in helping too. How best to get in touch? I've been using knex for 6 years, and we've been using it longer than that at gamingfactory.

**Comment by mercmobily** [comment]
@rubengmurray This is fantastic to hear. Please email me on tonymobily@gmail.com and I will get you on our Slack server!

**Comment by electriquo** [comment]
@mercmobily 

> [@electriquo](https://github.com/electriquo) I am organising a group of people willing to lend a hand to make sure knex is in active maintenance. Get in touch with me if you are willing to help!

Should I do that though the [email in this comment](https://github.com/knex/knex/issues/6154#issuecomment-3194797869)?

**Comment by rubengmurray** [comment]
> @rubengmurray This is fantastic to hear. Please email me on tonymobily@gmail.com and I will get you on our Slack server! 

Done 👍

**Comment by mercmobily** [comment]
@rubengmurray Are you still around?
